app/views/exclusion/WhatNextRescind.scala.html

```diff
@@ -45,12 +45,12 @@ <h1 id="title" class="govuk-panel__title">
 
     <div class="govuk-form-group">
         <p class="govuk-body" id="confirmation-p">
-            @messages("ExclusionRemoval.WhatNext.p1", name, messages("BenefitInKind.label." + uriInformation.iabdValueURLDeMapper(iabdType)), taxYearRange.cy)
+            @messages("ExclusionRemoval.WhatNext.p1", name, messages("BenefitInKind.label." + uriInformation.iabdValueURLDeMapper(iabdType)), taxYearRange.cy+"")
```

Comment:
can +"" be removed?

Reply:
If it is removed, the year is considered as number and page shows 2,022 instead of 2022.